docs/generate_petro_jobs_for_field_update.rst

```diff
@@ -0,0 +1,91 @@
+rms.generate_petro_jobs_for_field_update
+=========================================
+
+When running FMU project where field parameters for both facies and
+petrophysical properties is updated in ERT simultaneously,
+some adjustments are needed in the RMS project to support this type
+of workflow. It is necessary to have one petrosim job per facies.
+To simplify the work with the RMS project,
+the script *generate_petro_jobs_for_field_update* can be imported into
```

Comment:
I don't think you should refer to a script, but to a _function_ from fmu.tools. This paragraph need some rewriting

Reply:
Fixed